soroban-token-sdk/src/event.rs

```diff
@@ -22,6 +22,45 @@ impl Events {
         self.env.events().publish(topics, amount);
     }
 
+    pub fn transfer_with_muxed_address(&self, from: MuxedAddress, to: MuxedAddress, amount: i128) {
```

Comment:
I think this needs tweaking to change the from to an Address.

Reply:
Done.